internal/pubsub/rabbitpubsub/examples_test.go

```diff
@@ -26,7 +27,7 @@ import (
 
 const rabbitURL = "amqp://guest:guest@localhost:5672/"
 
-func Example_publish() {
+func Example() {
 	// Connect to RabbitMQ.
 	conn, err := amqp.Dial(rabbitURL)
 	if err != nil {
```

Comment:
Please verify that this test passes when you do have a local RabbitMQ running.

Consider adding some comments here explaining that the example code aborts early if rabbitURL isn't available.

Reply:
OK, I did something that makes it work as a test in both cases.